### PR TITLE
improve support by launching the app with corefoundationasyncio

### PR DESCRIPTION
**Pull request by typemytype**

### CODE REVIEW

**Comment by typemytype** [comment]
more info https://github.com/alberthier/corefoundationasyncio

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 setuptools>=45.2.0
 pyobjc
+CoreFoundationEventLoop
```
Comment (justvanrossum) [inline requirements.txt:3]: This should be `corefoundationasyncio`

Comment (typemytype) [reply requirements.txt:3]: oh offcourse...

**Comment by colinmford** [comment]
so psyched about this

**Comment by justvanrossum** [comment]
> so psyched about this

It's not a super easy paradigm to get into, but worth learning. FontGoggles uses it throughout to keep the UI responsive, if you'd like some non-trivial example code.

**Pull request merged**